.github/workflows/py-build.yml

```diff
@@ -1,4 +1,4 @@
-name: cell_census build
+name: Python cell_census build
```

Comment:
this file name is referenced in `release_process.md` - could you update the reference to point to the updated file name as part of this PR?